### ISSUE TITLE
Remove (abandonded) `container-interop/container-interop` dependency

### ISSUE THREAD

**Issue by bobdenotter** [Behat/Behat#1255]
Hi, 

Now that https://github.com/container-interop/container-interop/issues/94 has been done, composer update gives this mildly annoying message: 

```
Package container-interop/container-interop is abandoned, you should avoid using it. Use psr/container instead.
```

It's trivial in the greater scheme of things, but it'd be nice if this could be remedied.

**Comment by danepowell** [comment]
There's already an open issue for this from quite some time ago: https://github.com/Behat/Behat/issues/1110

It's unfortunate that this deprecation was known about for two years and nothing was done about it...

**Comment by Taluu** [comment]
As I already said on linked issue, this is considered to be a bc break, even though it's an acceptable one.

Ping @ciaranmcnulty, @stof and @jakzal

**Comment by pamil** [comment]
Fixed in #1281.
